src/components/Explore/TokenDetails/TokenDetail.tsx

```diff
@@ -243,19 +254,29 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
             </ClickFavorited>
           </TokenActions>
         </TokenInfoContainer>
-        <TokenPrice>${tokenPrice}</TokenPrice>
+        <TokenPrice>${tokenPrice.toFixed(2)}</TokenPrice>
         <DeltaContainer>
-          {deltaSign}
-          {tokenDelta}%
-          <ArrowCell>
-            {isPositive ? (
-              <ArrowUpRight size={16} color={theme.accentSuccess} />
-            ) : (
-              <ArrowDownRight size={16} color={theme.accentFailure} />
-            )}
-          </ArrowCell>
+          {tokenDelta ? (
+            <>
+              {deltaSign}
+              {tokenDelta.toFixed(2)}%
```

Comment:
this toFixed should happen before this imo. does the data come in with more decimals or something?

Reply:
it's just the result of division so it can and usually would I think, ill have it tranform it into a shortened string earlier

Reply:
right but we use toFixed(2) on the token price above too